src/main/kotlin/racingcar/Race.kt

```diff
@@ -0,0 +1,36 @@
+package racingcar
+
+fun main() {
+    val inputView = InputView()
+    val outputView = OutputView()
+
+    inputView.drawNumberOfCarInput()
+    val numberOfCar = readLine()!!.toInt()
```

Comment:
```suggestion
    val numberOfCar = readln().toInt()
```

readln을 사용해 볼까요? readln과 readLine의 차이점은 무엇일까요?

Reply:
반환형이 nullable이냐 아니냐 차이네요 수정하겠습니다